### PR TITLE
Updating Dashbook version

### PR DESCRIPTION
**Pull request by erickzanardo**
Updates Dashbook to the latest version and fix the theme issue

https://user-images.githubusercontent.com/835641/108129527-1631e380-708d-11eb-8523-65fe5aed09c3.mov

### CODE REVIEW

**Review by spydon — APPROVED** [review]
Nice, Dashbook ftw!

File: lib/main.dart [hunk 1]
```diff
@@ -14,7 +14,7 @@ import 'stories/tile_maps/tile_maps.dart';
 import 'stories/utils/utils.dart';
 
 void main() async {
-  final dashbook = Dashbook();
+  final dashbook = Dashbook(theme: ThemeData.dark());
```
Comment (luanpotter) [inline lib/main.dart:17]: can we add the title Flame Example here so it doesn't override the one in the HTML?

Comment (luanpotter) [reply lib/main.dart:17]: actually let me do that in a followup

**Review by luanpotter — APPROVED** [review]
LGTM

**Pull request merged**